### PR TITLE
fix(ci): e2e test timeout len and cache name

### PR DESCRIPTION
**Pull request by shawnyu5**
- Set timeout Len for E2E test to 30 mins
- Fix the key name which playwright browser is cached under

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -140,8 +140,7 @@ jobs:
       - name: Get installed Playwright version
         id: playwright-version
         run: |
-          echo "PLAYWRIGHT_VERSION=$(npm ls @playwright/test --json | jq --raw-output '.dependencies["@playwright/test"].version')"
-          echo $PLAYWRIGHT_VERSION >> $GITHUB_ENV
+          echo "PLAYWRIGHT_VERSION=$(npm ls @playwright/test --json | jq --raw-output '.dependencies["@playwright/test"].version')" >> $GITHUB_ENV
```
Comment (Eakam1007) [inline .github/workflows/ci.yaml:143]: I looked at the test run, and this works. I am confused why it wasn't working before. Could you please explain what changed here that made it work?

**Review by Eakam1007 — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (shawnyu5) [reply .github/workflows/ci.yaml:143]: Before, the cache under the name was `Linux-playwright-`, missing the version number. I just messed up injecting the env var into the github environment

Example [here](https://github.com/Seneca-CDOT/starchart/actions/runs/4118747950/jobs/7111645350#step:8:16)

**Review by SerpentBytes — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Eakam1007) [reply .github/workflows/ci.yaml:143]: Oh yea, I saw that. I am just confused about how changing this code fixes it

Comment (shawnyu5) [reply .github/workflows/ci.yaml:143]: Ah. It seems like gh needs the full shell syntax of assigning a value to a variable. ie `VAR=hello`. Before, I was doing creating the variable (`VAR=hello`), and putting the value of that variable(`hello`) into gh env. It's just `hello` floating around in there.

idk if this is a great way of explaining it

Comment (Eakam1007) [reply .github/workflows/ci.yaml:143]: Ah, I see. Thanks for the explanation!

**Pull request merged**